src/display/Plot/Parametric.tsx

```diff
@@ -4,10 +4,18 @@ import { Stroked } from "../Theme"
 import { useTransformContext } from "../../context/TransformContext"
 import { sampleParametric } from "./PlotUtils"
 
-export interface ParametricProps extends Stroked {
+// TODO: In 1-year's time from this change (written 6/26/2024),
+//          remove the Legacy interface and just have the new props interface (renamed, of course).
+//       Also, remove the `t` property at that time.
+//       The 1 year timeline gives consumers time to update their code.
```

Comment:
I don't base deprecations on time, I base them on semver. At some point there will be a 1.0 that will contain breaking changes, and will remove all currently-deprecated things.

Reply:
Updated.